c/compile_flags.txt

```diff
@@ -0,0 +1,18 @@
+-std=c99
+-pedantic
+-pipe
+
+-Wall
+-Wcast-qual
+-Wconversion
+-Werror=return-type
+-Werror=vla-larger-than=0
+-Wextra
+-Wmissing-prototypes
+-Wshadow
+-Wstrict-prototypes
+-Wwrite-strings
+
+-O2
+-march=native
+-mtune=native
```

Comment:
```suggestion
```
there's no reason to use these by default. it will only be confusing if anyone shares a locally built binary